contrib/kube-prometheus/hack/cluster-monitoring/deploy

```diff
@@ -16,9 +16,15 @@ until kubectl get prometheuses.monitoring.coreos.com > /dev/null 2>&1; do sleep
 until kubectl get alertmanagers.monitoring.coreos.com > /dev/null 2>&1; do sleep 1; printf "."; done
 echo "done!"
 
+# need to ensure that ServiceMonitors are registered before we can create th prometheus-operator ServiceMonitor
+find ${manifest_prefix}/manifests/prometheus-operator/ -type f -name prometheus-operator-service-monitor.yaml -exec kubectl apply -f {} \;
```

Comment:
Do I understand correctly, that this deploys only `prometheus-operator-service-monitor.yaml`? Why not deploy the file directly, instead of querying for it before via `find`?

Reply:
:man_facepalming: you're absolutely right